docs/enterprise-appstore/enterprise-app-store-profile.md

```diff
@@ -0,0 +1,272 @@
+:::
+
+:::info
+If Beta Channel Access is not set, the versions marked as the Beta channel will not be visible to anyone by default.
+
+If the Live Channel Access is not set, the versions marked as the Live channel will be visible to everyone by default.
+
+:::
+
+You can add defined user groups from your provider after configuration.
+
+<Screenshot url="https://cdn.appcircle.io/docs/assets/2812-entstore-okta-group-new.png" />
+
+:::tip 
+
+You can also update the logo of your Enterprise App Store profile by clicking on the logo icon. Please note that this will not affect your Enterprise Store login customization.
+
+:::
+
+<Screenshot url="https://cdn.appcircle.io/docs/assets/BE-4225-logo5.png" />
+
+## Binary Actions
+
+### Publish
+
+The Enterprise App Store module includes two channels: Beta and Live.
+
+- **Beta Channel**: This channel could be used for testing new updates, features, or changes before they are rolled out to all users. The beta channel is ideal for trialing updates without affecting the broader user base, ensuring that everything works as expected before moving to production.
+- **Live Channel**: The live channel is the production environment where all end users interact with the Enterprise App Store. This channel could contain the stable, fully tested binary versions.
+
+Apps can be sent to the Beta or Live channels by hitting the `...` button and then selecting the **Publish** menu.
+
+<Screenshot url="https://cdn.appcircle.io/docs/assets/BE-4225-beta1.png" />
+
+You can select the channel and write a summary and release notes for your release. When you hit the **Publish** button, that particular binary will be available to all beta users.
+
+<Screenshot url="https://cdn.appcircle.io/docs/assets/BE-4225-beta2.png" />
+
+<Screenshot url="https://cdn.appcircle.io/docs/assets/BE-4225-beta3.png" />
+
+You can send a version to the Live Channel in two ways:
+
+- Click the **Publish** button and select **Live** for the channel.
+- Click the ... button for any beta build and select **Go Live** from the menu.
+
+<Screenshot url="https://cdn.appcircle.io/docs/assets/BE-4225-golive.png" />
+
+:::info
+
+You can only send **one binary version** to the Live and Beta channels. When you send another Beta or Live binary version, the previous app version will be replaced from that channel.
+
+:::
+
+:::tip
+
+If you are part of a sub-organization, the app versions that you publish to the **Live** or **Beta** channel will be available on the Enterprise Store for users to view and download, just as they are for a root organization.
+
+:::
+
+When you publish a binary to the Live or Beta channel, the binary will be displayed with the corresponding channel tag. This information will also be shown in the profile header within the profile and on the profile card in the Enterprise App Store profile list.
```

Comment:
```suggestion
When a binary is published to the Live or Beta channel, it will be displayed with the corresponding channel tag. This information will also be shown in the profile header within the profile and on the profile card in the Enterprise App Store profile list.
```